test/e2e/send-eth-with-private-key-test/index.html

```diff
@@ -0,0 +1,17 @@
+<html>
```

Comment:
```suggestion
<html lang="en">
```

Reply:
Also we should make sure we have a doctype: `<!doctype html>` as the line above this one